src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,26 @@
-import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Movie {
+  title: string;
+  description?: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <>
+    <div className="movies">
+      {movies.map(movie => (
+        <div className="card" key={movie.imdbId}>
```

Comment:
```suggestion
    <ul className="movies">
      {movies.map(movie => (
        <li className="card" key={movie.imdbId}>
```
Краще використовувати семантичні теги. Це список, тому було б доречно використати ul

Reply:
Точно. дякую.
Трохи ввела в оману верстка,що була в App.